api/src/services/collections.ts

```diff
@@ -353,6 +361,14 @@ export class CollectionsService {
 			}
 		});
 
+		if (this.cache && env.CACHE_AUTO_PURGE) {
```

Comment:
here UpdateMany() does not have the optional 3rd argument `opts?: MutationOptions`, hence I couldn't do `opts?.autoPurgeCache !== false` for the if statement here. Is this ok or the UpdateMany() should've had this optional argument as well?

Reply:
Let's add the same opts object parameter and check here as well 👍🏻